extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -37,12 +37,27 @@
     @ConfigItem
     public Optional<String> applicationServer;
 
+    /**
+     * Do we check and wait for all topics from {@code topics} to be present,
+     * or do we ignore this check, default is false.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean ignoreTopics;
```

Comment:
Instead of adding that option, how about having a sentinel value for the timeout, such as 0, which disables the check? Feels a bit less complex in terms of the overall configuration surface to me, but of course both will work, so no blocker from my side.

Reply:
Now that `topics` is an optional config, maybe we can disable the check when `topics` is not configured.